packages/app-admin-ui/client/components/NoResults.js

```diff
@@ -42,16 +42,19 @@ export const NoResults = ({ currentPage, filters, list, search }) => {
     </NoResultsWrapper>
   );
 
+  if (currentPage !== 1) {
+    return pageDepthMessage;
+  }
+
   if (filters && filters.length) {
-    return currentPage !== 1 ? (
-      pageDepthMessage
-    ) : (
+    return (
       <NoResultsWrapper>
         No {list.plural.toLowerCase()} found matching the{' '}
         {filters.length > 1 ? 'filters' : 'filter'}
       </NoResultsWrapper>
     );
   }
+
   if (search && search.length) {
     return currentPage !== 1 ? (
```

Comment:
Can probably remove this one as well @jesstelford ?

Reply:
Yeah, looks like it 👍